src/wallet/rpc/spend.cpp

```diff
@@ -456,6 +457,53 @@ RPCHelpMan settxfee()
     };
 }
 
+RPCHelpMan settxfeerate()
+{
+    return RPCHelpMan{"settxfeerate",
+                "\nSet the transaction fee rate in " + CURRENCY_ATOM + "/vB for this wallet. Overrides the global -paytxfee command line parameter.\n"
+                "Can be deactivated by passing 0 as the fee. In that case automatic fee selection will be used by default.\n",
+                {
+                    {"amount", RPCArg::Type::AMOUNT, RPCArg::Optional::NO, "The transaction fee rate in " + CURRENCY_ATOM + "/vB"},
+                },
+                RPCResult{
+                    RPCResult::Type::BOOL, "", "Returns true if successful"
+                },
+                RPCExamples{
+                    HelpExampleCli("settxfee", "1")
```

Comment:
Examples copy+pasted but not updated

Reply:
thanks for the observation, will correct it